hr_employee_birth_name/readme/INSTALL.rst

```diff
@@ -0,0 +1,8 @@
+To install this module, you need to:
+
+1.  Clone the branch 11.0 of the repository https://github.com/OCA/hr
```

Comment:
This line should be updated, perhaps something a bit more general and not pointing to the specific version. Small detail, not a stopper.